src/Render.cc

```diff
@@ -116,6 +116,7 @@ namespace drafter {
                                                const refract::Registry& registry) {
 
         NodeInfoByValue<Asset> schema = std::make_pair(payload.node->schema, &payload.sourceMap->schema);
+        RenderFormat renderFormat = findRenderFormat(getContentTypeFromHeaders(payload.node->headers));
```

Comment:
Can this be moved to where it is needed, doing regex match every time even if we know that we don't needed is useless.